presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -664,6 +665,11 @@ public SystemSessionProperties(
                         OPTIMIZED_REPARTITIONING_ENABLED,
                         "Experimental: Use optimized repartitioning",
                         featuresConfig.isOptimizedRepartitioningEnabled(),
+                        false),
+                booleanProperty(
+                        SIMPLIFY_ARRAY_OPERATIONS,
+                        "Simplify and optimize array operations.",
```

Comment:
Probably indicate it will optimize `cardinality(filter(a, x->f(x))` expression in the session description will be more clear to user?

Reply:
Most users won't even understand what that means. This flag is only for any unforeseen bugs.